invenio/celery/__init__.py

```diff
@@ -143,15 +146,28 @@ def import_default_modules(self):
             self.autodiscover()
 
     def autodiscover(self):
-        """
-        Discover task modules named 'invenio.modules.*.tasks'
-        """
+        """Discover task modules named `invenio.modules.*.tasks`."""
         from invenio.celery import tasks
         self.task_modules.update(tasks.__name__)
         self.task_modules.update(
             mod.__name__ for mod in registry.tasks or ())
 
 
+# celery security config
+CELERY_SECURITY_SERIALIZER = 'json'
+CELERY_SECURITY_DIGEST = 'sha1'
+
+# try to load config before we have a valid application context
+try:
+    import invenio.base.config as conf
```

Comment:
I'm not sure you should be using invenio.base.config here. It's only default values and not the user configured ones.